source/common/decompressor/zlib_decompressor_impl.h

```diff
@@ -15,8 +15,12 @@ class ZlibDecompressorImpl : public Decompressor {
   ZlibDecompressorImpl();
 
   /**
-   * Sets buffer size for feeding data to the decompressor routines.
-   * @param chunk amount of memory reserved for the compressor output.
+   * Constructor that allows setting the size of decompressor's output buffer. It
+   * should be called whenever a buffer size different than the 4096 bytes, normally set by the
+   * default constructor, is desired. If memory is avaiable and it makes sense to output large
```

Comment:
avaiable typo snuck in again :)

Reply:
me and my typos.. sorry about that! Thanks @dnoe